pages/bitsofa/[slug].tsx

```diff
@@ -39,9 +39,8 @@ export const getStaticProps: GetStaticProps = async ({ params }) => {
             props: {
                 post,
             },
-            // Caches the article for 90 days
-            // 91st day onwards, new version will be fetched and cached for 90 days
-            revalidate: 60 * 60 * 24 * 90,
+            // Caches the article for 60s [testing]
+            revalidate: 60,
```

Comment:
revert this change. You don't need this on local though, it automatically refreshes everytime